src/main/java/teammates/ui/template/SoftDeletedFeedbackSessionsTable.java

```diff
@@ -0,0 +1,20 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SoftDeletedFeedbackSessionsTable {
```

Comment:
What is this for?

Reply:
This is a table that contains information of soft-deleted feedback sessions in the Recycle Bin. I renamed the file from `RecoveryFeedbackSessionsTable` to `SoftDeletedFeedbackSessionsTable`, so in `Files changed` page it shows that I deleted a file & created a new file.